scripts/jenkins/ardana/ansible/ardana-lint.yml

```diff
@@ -0,0 +1,27 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+
+- name: Run lint on ardana
+  hosts: "{{ ardana_env }}"
+  remote_user: ardana
+  gather_facts: True
+
+  tasks:
+    - block:
+        - include_role:
+            name: ardana_lint
```

Comment:
Will the lint rules be removed from ardana-dev-tools, it might be better to not have two copies that need to be kept in sync?

Reply:
That'd be a good thing to do as a follow up to this in the ci-squad. I'm not sure if it'd be better to remove or find a path to keep them in sync so that likely needs some thought.